### PR TITLE
Fixes links to OSS license texts

### LINKED ISSUE #440: License links lead to 404
## Task Headline

Many links on https://opensource.zalando.com/docs/resources/licensing/ lead to 404s.

For example https://tldrlegal.com/l/afl3 is now https://www.tldrlegal.com/license/academic-free-license-3-0-afl

### PR DESCRIPTION
**Pull request by bocytko**
tldrlegal.com changed their link scheme and old links lead to 404

Closes #440

### CODE REVIEW

**Comment by bocytko** [comment]
@szafranek

**Comment by gchudnov** [comment]
:+1:

**Comment by bocytko** [comment]
@gchudnov thanks. I believe this repo requires GH native approvals from a maintainer on the PRs ;/

**Review by gchudnov — APPROVED** [review]

**Review by szafranek — APPROVED** [review]

**Pull request merged**
